x-pack/plugins/endpoint/public/embeddables/resolver/view/process_event_dot.tsx

```diff
@@ -114,14 +114,20 @@ export const ProcessEventDot = styled(
         [left, magFactorX, top]
       );
 
+      /**
+       * Type in non-SVG components scales as follows:
+       *  (These values were adjusted to match the proportions in the comps provided by UX/Design)
+       *  18.75 : The smallest readable font size at which labels/descriptions can be read. Font size will not scale below this.
+       *  12.5 : A 'slope' at which the font size will scale w.r.t. to zoom level otherwise
+       */
+      const scaledTypeSize = useMemo(() => {
+        return magFactorX > 1 ? 18.75 + 12.5 * (magFactorX - 1) : 18.75;
```

Comment:
might help make this look less magic numbery to pull these values out into variables, i.e. `const minimumFontSize = 18.75;` etc. Also useMemo is probably not needed, and the overhead of using it is probably less performant than a small calculation like this

Reply:
https://github.com/elastic/kibana/pull/62958/commits/b0f29c1cff6512da6a89919494c23d6c573a70d6